app/views/toggle/_temporary_toggle_row.html.erb

```diff
@@ -0,0 +1,27 @@
+<td class="details">
```

Comment:
New partial for the temporary toggle row, as it now differs from a permanent toggle row.